Symfony/CS/Fixer/Contrib/UselessReturnFixer.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ */
+final class UselessReturnFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        for ($index = 0, $count = $tokens->count(); $index < $count; ++$index) {
+            if (!$tokens[$index]->isGivenKind(T_FUNCTION)) {
+                continue;
+            }
+
+            $index = $functionOpen = $tokens->getNextTokenOfKind($index, array('{'));
+            $this->fixFunction($tokens, $functionOpen, $tokens->findBlockEnd(Tokens::BLOCK_TYPE_CURLY_BRACE, $functionOpen));
+        }
+
+        return $tokens->generateCode();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDescription()
+    {
+        return 'There should not be an empty return statement at the end of a function.';
```

Comment:
Yeah, we all agree here: nothing to change in this fixer and no need for another fixer.